tests/app/service/test_rest.py

```diff
@@ -1059,6 +1059,28 @@ def test_get_all_notifications_for_service_including_ones_made_by_jobs(
         assert response.status_code == 200
 
 
+def test_get_only_api_created_notifications_for_service(
+    notify_api,
+    notify_db,
+    notify_db_session,
+    sample_service
+):
+    with notify_api.test_request_context(), notify_api.test_client() as client:
+        with_job = sample_notification_with_job(notify_db, notify_db_session, service=sample_service)
+        without_job = create_sample_notification(notify_db, notify_db_session, service=sample_service)
+
+        auth_header = create_authorization_header()
+
+        response = client.get(
+            path='/service/{}/notifications?include_jobs=false'.format(sample_service.id),
+            headers=[auth_header])
+
+        resp = json.loads(response.get_data(as_text=True))
+        assert len(resp['notifications']) == 1
+        assert resp['notifications'][0]['to'] == without_job.to
```

Comment:
they'll probably both have the same phone number - can you check their `id` instead?